kentikapi/client_integration_test.go

```diff
@@ -45,7 +45,8 @@ func TestClient_GetUserWithRetries(t *testing.T) {
 				testutil.NewErrorHTTPResponse(http.StatusBadGateway),
 				{StatusCode: http.StatusOK, Body: "invalid JSON"},
 			},
-			expectedError: true,
+			expectedError:   true,
+			expectedTimeout: true,
```

Comment:
This subtest is broken now - it does not verify any retry. Please remember that the purpose of tests is to verify particular use-case of the client. Not to pass. Clearly `A Timeout of zero means no timeout.` docstring from WithTimeout() func is not true and should be removed.
Please use default timeout in subtests, that do not need to modify the timeout. It can be done similarly to options in TestClient_GetAgentWithRetries

Reply:
Done